client/lib/support/support-user/test/localstorage-bypass.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
```

Comment:
The usage here is fine, but we also have a number of test helpers now in our single test runner, including: https://github.com/Automattic/wp-calypso/tree/master/test/test/helpers/use-sinon